app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    if (command[:3]) != "mv " or command.count(" ") != 2:
```

Comment:
The condition `(command[:3]) != "mv "` is correct for checking the command prefix, but the check `command.count(" ") != 2` might not be sufficient if there are extra spaces in the command. Consider using `command.split()` to handle multiple spaces more robustly.